parsers.js

```diff
@@ -0,0 +1,23 @@
+export async function jsonLdParser () {
+  const JsonLdParser = (await import('@rdfjs/parser-jsonld')).default
+
+  return new JsonLdParser()
+}
+
+export async function n3Parser () {
+  const N3Parser = (await import('@rdfjs/parser-n3')).default
+  return new N3Parser()
+}
+
+export async function rdfXmlParser () {
+  const module = await import('rdfxml-streaming-parser')
+
+  let RdfXmlParser
+  if (module.default) {
+    RdfXmlParser = module.default.RdfXmlParser
+  } else {
+    RdfXmlParser = module.RdfXmlParser
+  }
```

Comment:
I would need to look that up, but I think that in some cases the import would have two forms. Maybe that was depending on the version rdf parser used but maybe it's differences in bundler handling...